SDKs/PowerShell/JumpCloud.SDK.V2/test/Set-JcSdkPolicyGroupMember.Tests.ps1

```diff
@@ -16,7 +16,9 @@ while(-not $mockingPath) {
 }
 Describe 'Set-JcSdkPolicyGroupMember' -Tag:(""){
     It 'SetExpanded' {
-        { Set-JcSdkPolicyGroupMember -GroupId:($global:PesterTestPolicyGroup.Id) -Id:($global:PesterTestUser.Id) -Op 'add' } | Should -Not -Throw
+        $PesterTestPolicyGroup = New-JcSdkPolicyGroup -Name "TestGroupMember"
```

Comment:
You'll need to remove this group at the end of your test in order for this to pass next time